app/services/idv/in_person_config.rb

```diff
@@ -6,7 +6,7 @@ def self.enabled_for_issuer?(issuer)
       if issuer.nil?
         enabled_without_issuer?
       else
-        enabled_issuers.include?(issuer)
+        ServiceProvider.find_by(issuer: issuer)&.in_person_proofing_enabled || false
```

Comment:
am i understanding this correctly - we're finding a service provider by issuer who has in person proofing enabled or using false (issuer was not found?). Is "false" allowing for a situation where an issuer is provided but either does not exist or does not have in person proofing enabled?

Reply:
> Is "false" allowing for a situation where an issuer is provided but either does not exist or does not have in person proofing enabled?

Yes, this allows for both situations.

```
(query logs removed below)

$ rails console   
Loading development environment (Rails 7.0.3.1)
[1] pry(main)> ServiceProvider.find_by(issuer: "doesnotexist")&.in_person_proofing_enabled || false
=> false
[2] pry(main)> ServiceProvider.find_by(issuer: "urn:gov:gsa:openidconnect.profiles:sp:sso:gsa:dashboard")&.in_person_proofing_enabled || false
=> false
[3] pry(main)> ServiceProvider.find_by(issuer: "urn:gov:gsa:openidconnect:sp:sinatra")&.in_person_proofing_enabled || false
=> true
```